src/main/java/org/elasticsearch/indices/cluster/IndicesClusterStateService.java

```diff
@@ -445,6 +469,7 @@ private boolean processMapping(String index, MapperService mapperService, String
             }
         } catch (Throwable e) {
             logger.warn("[{}] failed to add mapping [{}], source [{}]", e, index, mappingType, mappingSource);
```

Comment:
I kept it around since at this level we have the data about the mapping type and the source